<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imgviewer_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>441</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Image Viewer</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomIn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>570</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom In(+25%)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOut">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>610</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Out(-25%)</string>
    </property>
   </widget>
   <widget class="QLabel" name="local_mousepos_label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>680</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Mouse Position x: y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="seedpos_label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>710</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seed Position x: y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="global_mousepos_label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>650</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Global Mouse Position x: y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maskviewer_label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>381</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Mask Viewer</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImage_Scissor">
    <property name="title">
     <string>Image Scissor</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_Contour"/>
    <addaction name="actionSave_Mask"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionPixel_Node"/>
    <addaction name="actionCost_Graph"/>
    <addaction name="actionMinimum_Path"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionSet_Seed"/>
    <addaction name="actionDelete_Latest_Seed"/>
   </widget>
   <addaction name="menuImage_Scissor"/>
   <addaction name="menuTool"/>
   <addaction name="menuAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSave_Contour">
   <property name="text">
    <string>Save Contour</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Mask">
   <property name="text">
    <string>Save Mask</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionCost_Graph">
   <property name="text">
    <string>Cost Graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionMinimum_Path">
   <property name="text">
    <string>Minimum Path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSet_Seed">
   <property name="text">
    <string>Set Seed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionDelete_Latest_Seed">
   <property name="text">
    <string>Delete Latest Seed</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPixel_Node">
   <property name="text">
    <string>Pixel Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
